### DISCUSSION TITLE
Duplicate fork activity

### DISCUSSION THREAD

**Discussion by xhd2015** [community/community#119627] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Another github user forked my repo once, but shown twice on the feed

The repository: https://github.com/xhd2015/xgo
Forked: https://github.com/kilianc/xgo

![image](https://github.com/community/community/assets/14964938/eb2c7098-b835-4011-a166-91bd780caa46)

**Comment by olekstomek** [comment]
I have the same behavior for the starred action.
![two_the_same_repositories](https://github.com/community/community/assets/26818304/b417c84c-24c0-4d90-aca8-e3de277d2714)

**Reply by LiteBrite82** [reply]
This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.

**Comment by stwala** [comment]
Cached data or cookies could be causing the duplication problem. Clear your browser's cache and cookies, then revisit your GitHub feed to check if the issue persists.

**Reply by olekstomek** [reply]
I tried in another browser. Then clear cookies and local storage for GitHub pages in this browser. The problem is the same, I still see duplicate information.
